rcl/test/CMakeLists.txt

```diff
@@ -65,10 +65,6 @@ function(test_target_function)
 
   rcl_add_custom_gtest(test_get_node_names${target_suffix}
     SRCS rcl/test_get_node_names.cpp
-    ENV
-      ${extra_test_env}
```

Comment:
If you keep several rmw for now, please also pass the RMW_IMPLEMENTATION you want to test (by keeping line 68-69)